spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "rails_helper"
+
+RSpec.describe "guests show page" do
+
+# Story 1
+# Guest Show
+# As a visitor
+# When I visit a guest's show page
+# I see the guest's name
+# And I see a list of all the rooms they've stayed in
+# including the room's suite, nightly rate, 
+# and the name of the hotel that it belongs to.
+
+  it "displays the guest's name" do
+    hotel_1 = Hotel.create!(name: "Aspen Inn", location: "Aspen" )
```

Comment:
I'd recommend using a before each so that you don't have to copy and paste the test set up in each test case